scheduler-server/src/test/java/gov/nasa/jpl/aerie/scheduler/server/services/TypescriptCodeGenerationServiceTest.java

```diff
@@ -49,6 +52,7 @@ void testCodeGen() {
                  + "      return { activityType: 'PeelBanana', args };\n"
                  + "    },\n"
                  + "}\n"
+                 + "export const ActivityTypes = ActivityType;\n"
```

Comment:
In fact, we don't even need to export a const here, we can just export then enum directly

Reply:
True - I guess I was trying to mirror the pattern we used for `ActivityTemplates`, where the plural is used for the name of the map, and the singular is used for the name of the type. Not sure how important that consistency is though.

Reply:
Fair point! I agree